index.html

```diff
@@ -173,5 +262,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2 class="why-choose-us-head">WHY CHOOSE US</h2>
+            <p class="why-choose-us-text">Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics</p>
+        </div>
+        <div class="grid-content">
+            <div class="grid-item">
+                <div class="grid-item-header">
+                <img class="grid-icons" src="img/icons/briefcase.svg" alt="briefcase icon">
+                <h5 class="grid-header-text">Digital Marketing</h5>
+                </div>
+                <p class="grid-item-text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="grid-item">
```

Comment:
I suggest you to rethink the grid-item namings.
Let's say what kind of data is that? If you take a closer look at the rest of the design you will notice that there are more of these cards.

So I suggest you to give each of the cards class card and use it for all sections in future.
The parent, the grid one contains cards. So you could give it a class called cards or cards-grid if you want to specify word "grid" there.
Inside each card you have "card header" and "card content". Here are the class names for them: card__header and card__content

Inside header you have icon and title.
So header__icn and header__title sound logical for their class names.

Basically you've chosen content agnostic class names as well. And you could use them for future elements. But "grid-item" doesn't tell us what kind of element is there. But "card" tells us that the element is a card and will look like a card. (it's a best practice to call cards elements like these)